kb/eekb/agent_of_finding_sc_elements_without_any_main_identifier/agent_component/lib_component_ui_menu_file_for_finding_sc_elements_without_any_main_identifier/lib_component_ui_menu_file_for_finding_sc_elements_without_any_main_identifier.scs

```diff
@@ -0,0 +1,25 @@
+lib_component_ui_menu_file_for_finding_sc_elements_without_any_main_identifier
+=> nrel_main_idtf:
+	[Компонент библиотеки. Команда пользовательского интерфейса для поиска sc-элементов, имеющих системный идентификатор, но не имеющих ни одного основного]
+	(* <- lang_ru;; *);
+	[Library component. User interface command for finding sc-elemnts without any main identifier]
```

Comment:
... for finding sc-elements with system identifier but without any main identifier